README.md

```diff
@@ -1,122 +1,128 @@
+  greater_or_equal_than 300, :amount
+end
 
-## Example
-```ruby
-ruleset = Ruy::RuleSet.new
+gifter.outcome 7 do
+  greater_or_equal_than 100, :amount
+end
 
-# RuleSets can use :any and :all to do AND and OR groupings of conditions.
+gifter.outcome 3
 
-ruleset.any do
+gifter.fallback 0
+```
 
-    # RuleSets evaluate against Context variables.
-    # Equality, scalarity (less than, greater
-    # than or equal to, etc), and inclusion/exclusion
-    # are supported
+RuleSets are evaluated against a context (the `Hash` being passed to `#call`) and return the first outcome that matches.
 
-    between 18, 22, :age
-    eq 'University of Texas', :school
-end
+``` ruby
+gifter.call(day_of_week: :friday, amount: 314)
 
-# RuleSets can have multiple outcomes when
-# conditions obtain in a certain Context.
+# => 7
+```
 
-ruleset.outcome "I matched!"
+If no outcomes match, the default one is returned.
+``` ruby
+gifter.call(day_of_week: :friday, amount: 99)
 
-ruleset.(age: 20) #=> "I matched!"
-ruleset.(age: 20, school: "University of Alabama" #=> "I matched!"
-ruleset.(age: 16, school: 'University of Texas' #=> "I matched!"
-ruleset.(age: 10, school: 'University of Alabama') #=> nil
+# => 3
 ```
 
-## Concepts
+If conditions are not met, the fallback value is returned.
+``` ruby
+gifter.call(day_of_week: :monday, amount: 124)
 
-Ruy at its core is about evaluating something against a rule. What's a rule? And what's "something"? Let's take a look.
-
-### Rules
+# => 0
+```
 
-A `Rule` is a set of conditions. Ruy has a number of builtin conditions:
+## Key concepts
 
-* Equal (**:eq**)
-* Assert (**:assert**)
-* Between (**:between**)
-* Not (**:except**)
-* Inclusion (**:include** and **:included**)
-* Less than/Greater than (**:less\_than**, **:less\_than\_or\_equal\_to**, **:greater\_than**, **:greater\_than\_or\_equal\_to**)
+Ruy at its core is about evaluating a set of conditions against a context in order to return a result.
 
-Rules can also group conditions arbitrarily as logical ANDs (**:all**) or OR (**:any**).
+### Conditions
 
-#### RuleSet-level lazy values
+A condition evaluates the state of the context.
 
-Rules can have lazy values set for them. The context must provide a proc which is evaluted only once the first time the value is needed to evaluate a rule. The result returned by the proc application is memoized and used to evaluate subsequent rules.
+Available conditions:
 
-```ruby
-rule.let :some_expensive_calculation
-```
-#### Applying Rules
+ - all *All of the nested conditions must suffice*
+ - any *At least one of its nested conditions must suffice*
+ - assert *A context value must be truish*
+ - between *Evaluates that a context value must belong to a specified range*
+ - cond *At least one slice of two nested conditions must suffice*
+ - day_of_week *Evaluates that a Date/DateTime/Time weekday is matched*
+ - eq *Tests a context value for equality*
+ - except *Evalutes that a any of the nested conditions don't suffice*
```

Comment:
"Evaluates that all of the nested condition don't suffice"

Remember that this condition also takes an attribute to evaluate from the context and check that is not equal to a passed value.